src/clr-angular/data/tree-view/tree-node.ts

```diff
@@ -40,26 +48,36 @@ import { TREE_FEATURES_PROVIDER, TreeFeaturesService } from './tree-features.ser
     { provide: LoadingListener, useExisting: IfExpandService },
   ],
   animations: [
-    trigger('childNodesState', [
-      state('expanded', style({ height: '*', 'overflow-y': 'hidden' })),
-      state('collapsed', style({ height: 0, 'overflow-y': 'hidden' })),
-      transition('expanded <=> collapsed', animate('0.2s ease-in-out')),
+    trigger('toggleChildrenAnim', [
+      transition('void => *', [
+        style({ display: 'block', height: 0, 'overflow-y': 'hidden' }),
+        animate(200, style({ height: '*' })),
+      ]),
+      transition('* => void', [
+        style({ display: 'block', 'overflow-y': 'hidden' }),
+        animate(200, style({ height: 0 })),
+      ]),
     ]),
   ],
-  host: { '[class.clr-tree-node]': 'true' },
+  host: {
+    '[attr.role]': '"treeitem"',
```

Comment:
If we have the role of `treeitem` on the `contentContainer`, I don't think the host should have it also..? Ultimately, the element that receives focus should have both the role and all the `aria-` attributes assigned to it.

Reply:
Good question. This was a fix to the Safari + VO. 
I had this previously without `role="treeitem"` on the `clr-tree-node` host element as follows:
```
<clr-tree-node>
  <div class="clr-tree-node-content-container" role="treeitem">...</div>
  <div class="clr-treenode-children" role="group">...</div>
<clr-tree-node>
```
However, the Safari +VO combination wasn't treating the `group` and `treeitem` as parts of the same `treeitem`. Then, after reading some articles, I tried to solve with `aria-owns` attribute like this:
```
<clr-tree-node>
  <div class="clr-tree-node-content-container" role="treeitem" aria-owns="clr-uniq-id-00">...</div>
  <div class="clr-treenode-children" role="group" id="clr-uniq-id-00">...</div>
<clr-tree-node>
```
Still the same result. The Safari + VO didn't tree them as one peace. Finally, I tried this:

```
<clr-tree-node role="treeitem">
  <div class="clr-tree-node-content-container" role="treeitem">...</div>
  <div class="clr-treenode-children" role="group">...</div>
<clr-tree-node>
```

Now, The Safari + VO started working the same as it worked in the spec guideline examples.